<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookingDetails</class>
 <widget class="QDialog" name="bookingDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Flug">
        <attribute name="icon">
         <iconset>
          <normaloff>flight.png</normaloff>
          <normalon>:/flight.png</normalon>flight.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Flugbuchung</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>ID:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="flugID">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Airline:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="flugAirline"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>From Date:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="flugFromDate"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>To Date:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="flugToDate"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>From:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="flugFromDest"/>
             </item>
             <item>
              <widget class="QLineEdit" name="fromName"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_18">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>To:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="flugToDest"/>
             </item>
             <item>
              <widget class="QLineEdit" name="toName"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_32">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Class:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="flugClass"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_34">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Price:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="flugPrice"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Zug">
        <attribute name="icon">
         <iconset>
          <normaloff>train.jpg</normaloff>
          <normalon>:/train.jpg</normalon>train.jpg</iconset>
        </attribute>
        <attribute name="title">
         <string>Zugbuchung</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>ID:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="zugID">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="label_22">
                 <property name="text">
                  <string>Class:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="zugClass"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>From Date:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="zugFromDate"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <item>
                <widget class="QLabel" name="label_24">
                 <property name="text">
                  <string>Price</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="zugPrice"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>To Date:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="zugToDate"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>From:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="zugFrom"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>To:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="zugTo"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>Departure:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="zugDeparture"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="label_21">
                 <property name="text">
                  <string>Arrival:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="zugArrival"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Connecting Stations:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="zugCS">
               <property name="editTriggers">
                <set>QAbstractItemView::AllEditTriggers</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Mietwagen">
        <attribute name="icon">
         <iconset>
          <normaloff>car.png</normaloff>
          <normalon>:/car.png</normalon>car.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Mietwagenbuchung</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>881</width>
           <height>571</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="5" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_23">
            <item>
             <widget class="QLabel" name="label_30">
              <property name="text">
               <string>Pickup Location:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="carPickUp"/>
            </item>
           </layout>
          </item>
          <item row="7" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_25">
            <item>
             <widget class="QLabel" name="label_32">
              <property name="text">
               <string>Price:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="carPrice"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Class</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="carClass"/>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_24">
            <item>
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>Return Location:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="carReturn"/>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_20">
            <item>
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>Company:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="carCompany"/>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>From Date</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="carFromDate"/>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <item>
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>To Date:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="carToDate"/>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="carID">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="Hotel">
        <attribute name="icon">
         <iconset>
          <normaloff>hotel.png</normaloff>
          <normalon>:/hotel.png</normalon>hotel.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Hotelbuchung</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="3" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_29">
             <item>
              <widget class="QLabel" name="label_39">
               <property name="text">
                <string>Town:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hotelTown"/>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_28">
             <item>
              <widget class="QLabel" name="label_40">
               <property name="text">
                <string>From Date:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hotelFromDate"/>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_26">
             <item>
              <widget class="QLabel" name="label_42">
               <property name="text">
                <string>Price:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hotelPrice"/>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_27">
             <item>
              <widget class="QLabel" name="label_41">
               <property name="text">
                <string>To Date:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hotelToDate"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_30">
             <item>
              <widget class="QLabel" name="label_38">
               <property name="text">
                <string>Class:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hotelClass"/>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_33">
             <item>
              <widget class="QLabel" name="label_34">
               <property name="text">
                <string>ID:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hotelID">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_31">
             <item>
              <widget class="QLabel" name="label_36">
               <property name="text">
                <string>Hotel:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="hotelHotel"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>bookingDetails</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>bookingDetails</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
